### PR TITLE
Add recipe for ford

### PR DESCRIPTION
**Pull request by awvwgk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mfansler** [comment]
FYI: The check reporting appears not to be syncing, but [the CI pipelines fail](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=432880&view=results), with the message that the package `md-environ` is not available.

**Comment by awvwgk** [comment]
@conda-forge/staged-recipes This recipes are ready for review.

File: recipes/ford/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/FORD-{{ version }}.tar.gz
+  sha256: 24c5868b02a584464c74acfa48b673a8a27ff84c7dc9beda5a37acd4cd3671e0
+
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/ford/meta.yaml:14]:
```suggestion
  number: 0
  noarch: python
```

Comment (awvwgk) [reply recipes/ford/meta.yaml:14]: What about `importlib-metadata`? This needs a Python version selector.

Comment (xhochy) [reply recipes/ford/meta.yaml:14]: You could remove the selector, installing it on py>=38 should be a no-op and `noarch` packages are generally easier to maintain.

**Pull request merged**